app.js

```diff
@@ -1,11 +1,64 @@
 /* Imports */
+const shell1Button = document.getElementById('shell1-button');
+const shell2Button = document.getElementById('shell2-button');
+const shell3Button = document.getElementById('shell3-button');
+
+const shell1Container = document.getElementById('shell1-container');
+const shell2Container = document.getElementById('shell2-container');
+const shell3Container = document.getElementById('shell3-container');
+
+const totalEl = document.getElementById('total');
+const lossesEl = document.getElementById('losses');
+const winsEl = document.getElementById('wins');
+
+let correctGuesses = 0;
+let totalGuesses = 0;
 
 /* Get DOM Elements */
+shell1Button.addEventListener('click', () => {
+    const correctSpot = getRandomHidingSpot();
+    handleGuess('shell1', correctSpot);
+    console.log(correctSpot);
```

Comment:
feel free to delete these console.log statements once you've verified functionality